integration/tests/handshake.test.ts

```diff
@@ -72,7 +73,7 @@ test.describe('Client handshake @generic', () => {
     await new Promise<void>(resolve => jwksServer.close(() => resolve()));
   });
 
-  test('Test standard signed-in - dev', async () => {
```

Comment:
My IDE automatically deleted all of these `Test` prefixes, citing this:

```
ESLint: should not have duplicate prefix(playwright/valid-title)
```

I can add them back if we like them though!